src/assets/wise5/components/peerChat/peer-chat-student/peer-chat-student.component.ts

```diff
@@ -280,16 +270,21 @@ export class PeerChatStudentComponent extends ComponentStudent {
     }
   }
 
-  studentResponseChanged(): void {
-    this.isSubmitEnabled = this.messageText.length > 0;
+  getAvatarColor(workgroupId): string {
+    return this.ConfigService.getAvatarColorForWorkgroupId(workgroupId);
   }
 
-  studentKeyPressed(event: any): void {
-    if (event.keyCode === 13) {
-      event.preventDefault();
-      if (this.isSubmitEnabled) {
-        this.submitStudentResponse();
-      }
+  getDisplayNames(workgroupId: number): string {
+    // TODO: duplciate code in DiscussionService (setUsernames); extract to ConfigService?
```

Comment:
write up issue?

Reply:
Yep, I'll write this up. Function now uses the workgroupId param correctly.